pallets/nfts/src/lib.rs

```diff
@@ -465,21 +498,31 @@ pub mod pallet {
 		/// a `delegate`.
 		TransferApproved {
 			collection: T::CollectionId,
-			item: T::ItemId,
+			item: Option<T::ItemId>,
```

Comment:
Note: Tested in:

- approving_multiple_works: https://github.com/r0gue-io/pop-node/pull/387/files#r1881289915